build-kite/vm_scripts/start-agent.sh

```diff
@@ -44,9 +44,11 @@ $AS_AGENT git config --global push.default simple
 # Add pipeline specific secrets to environment
 HINT_CODECOV=$(vault read -field=token secret/hint/codecov)
 MINT_CODECOV=$(vault read -field=token secret/mint/codecov)
+ORDERLYWEB_CODECOV=$(vault read -field=token secret/vimc/orderly-web/codecov)
```

Comment:
We have two vaults; one is vimc-only and one is everything else. At some point we will migrate all of the vimc vault into the centre one, below `vimc/` but preserving the structure. However, even with that wrinkle, this looks like a good place to put it.